hack/aks/Makefile

```
@@ -43,6 +48,23 @@ azcfg: ## Set the $AZCLI to use aks-preview
 	@$(AZCLI) extension add --name aks-preview --yes
 	@$(AZCLI) extension update --name aks-preview
 
+ip:
+	$(AZCLI) network public-ip create --name $(IP_PREFIX)-$(CLUSTER)-$(IPVERSION) \
+		--resource-group $(GROUP) \
+		--allocation-method Static \
+		--ip-tags $(IP_TAG) \
+		--location $(REGION) \
+		--sku Standard \
+		--tier Regional \
+		--version IP$(IPVERSION)
+
+ipv4:
+	@$(MAKE) ip IPVERSION=v4
+
+ipv6:
+	@$(MAKE) ip IPVERSION=v6
```

nit: this looks excessive. We could possibly leverage https://www.gnu.org/software/make/manual/make.html#Target_002dspecific 
```suggestion
ipv4: IPVERSION = v4
ipv4: ip

ipv6: IPVERSION = v6
ipv6: ip
```

---

@rbtr, opinions?